src/backlinks/handler.js

```diff
@@ -139,13 +139,67 @@ export default async function auditBrokenBacklinks(message, context) {
       auditResult,
     };
 
-    await dataAccess.addAudit(auditData);
+    const audit = await dataAccess.addAudit(auditData);
     const data = {
       type,
       url: site.getBaseURL(),
       auditContext,
       auditResult,
     };
+
+    const opportunities = await dataAccess.Opportunity.allBySiteIdAndStatus(siteId, 'NEW');
+    let brokenBacklinksOppty = opportunities.find((oppty) => oppty.getType() === 'broken-backlinks');
```

Comment:
if this is a frequent access pattern i could also introduce a relevant index and a `allBySiteIdAndTypeAndStatus` method. WDYT?

Reply:
yes I can add it

Reply:
still pending, ElectroDB does not allow readme properties to be part of the composite. Solution coming with next PR